doc/releases/changelog-dev.md

```diff
@@ -20,34 +19,32 @@
   [(#2874)](https://github.com/PennyLaneAI/pennylane/pull/2874)
   [(#2644)](https://github.com/PennyLaneAI/pennylane/pull/2644)
 
-  The resource estimation algorithms are implemented as classes inherited from the `Operation`
-  class. The number of non-Clifford gates and logical qubits for implementing each algorithm can be
-  estimated by initiating the class for a given system. For the first quantization algorithm, the 
-  number of plane waves, number of electrons and the unit cell volume (in atomic units) are needed
-  to initiate the `FirstQuantization` class. The resource can then be estimated as
+  The new `qml.resource` module allows you to estimate the number of non-[Clifford gates](https://en.wikipedia.org/wiki/Clifford_gates) 
+  and logical qubits needed to implement quantum phase estimation algorithms 
+  for simulating materials and molecules. This includes support for quantum 
+  algorithms using first and second quantization with specific bases:
+
+  - First quantization using a plane-wave basis via the `FirstQuantization` class:
 
   ```python
   import pennylane as qml
   from pennylane import numpy as np
   
   n = 100000        # number of plane waves
   eta = 156         # number of electrons
-  omega = 1145.166  # unit cell volume
+  omega = 1145.166  # unit cell volume in atomic units
   
   algo = FirstQuantization(n, eta, omega)
-  
-  # print the number of non-Clifford gates and logical qubits
-  print(algo.gates, algo.qubits)
   ```
   
   ```pycon
+  >>> print(algo.gates, algo.qubits)
   1.10e+13, 4416
   ```
-  
-  For the second quantization algorithm, the one- and two-electron integrals are needed to initiate
-  the `DoubleFactorization` class which creates a double-factorized Hamiltonian and computes the
-  number of non-Clifford gates and logical qubits for simulating the Hamiltonian:
 
+  - Second quantization with a double-factorized Hamiltonian via the 
+  `DoubleFactorization` class: 
```

Comment:
Not sure what double-factorized means here 🤔

Reply:
It refers to a Hamiltonian built from a low-rank factorization of the electron-integral tensor. The factorization has two steps and allows representing the large tensor with a truncated series of smaller matrices and their diagonalization. The details are explained [here](https://arxiv.org/abs/2007.14460).

Reply:
Thank you @soranjh 💯

Reply:
Would this reference suffice?

https://www.nature.com/articles/s41534-021-00416-z

I'm mainly looking for some explanation of "double-factorized" specifically.